### PR TITLE
Log more detail in error parsing X-Socrata-Fuse-Columns

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

File: query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala [hunk 1]
```diff
@@ -410,7 +410,10 @@ class QueryResource(secondary: Secondary,
   private def parseFuseColumnMap(s: String): Map[String, String] = {
     s.split(';')
      .map { item => item.split(',') }
-     .map { case Array(a, b) => (a, b) }
+     .map { case Array(a, b) => (a, b)
+            case _ =>
```
Comment (PiSoup) [inline query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala]: Maybe also throw the actual exception too.

Comment (PiSoup) [reply query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala]: Or include the message from it on the new one.

File: query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala [hunk 2]
```diff
@@ -410,7 +410,10 @@ class QueryResource(secondary: Secondary,
...
+               throw new Exception(s"X-Socrata-Fuse-Columns parse error $s.")
+     }
```
Comment (gerald-alva) [inline query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala]: Will this stop the com.socrata.http.client.exceptions.UnexpectedContentType: Wanted content-type application/json; got text/html error we see in SF?

File: query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala (hunk 1 above)
Comment (chitang) [reply query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala]: The actual exception is no match case in scala

File: query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala (hunk 2 above)
Comment (chitang) [reply query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala]: no, this isn't trying to fix the problem.  just want to get more information out.  I am hoping that the problem is obvious once we see the value in X-Socrata-Fuse-Columns

**Pull request merged**